### PR TITLE
blog: mysql to email

### PR DESCRIPTION
**Pull request by Michaelg22**
### Related Issue

Solving Issue: #xxx

### PR Type

- [ ] Fix typos or format(punctuation, space, indentation, code block, etc.)
- [ ] Update inappropriate or misunderstanding content
- [ ] Add missing content or new documents
- [ ] Delete useless content or outdated documents
- [ ] Localize or translate documents
- [ ] Deal with GitHub action files and scripts
- [ ] Deal with chores

### CODE REVIEW

File: blog/2023-02-19-mysql-email-blog/index.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+### Step 1: Deploy Vanus on the Playground
+- Go to Vanus Playground, and click “Continue with GitHub”
+
+ ![](img/img.png) 
+
+- Wait until the K8s environment is ready (usually less than 1 min).
+  img
+- Install Vanus by typing the following command:
+  kubectl apply -f https://download.linkall.com/all-in-one/v0.6.0.yml
```
Comment (ehisakhile) [inline blog/2023-02-19-mysql-email-blog/index.md:84]:
```shell
kubectl apply -f https://download.linkall.com/all-in-one/v0.6.0.yml
```

File: blog/2023-02-19-mysql-email-blog/index.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+table_include: [ "TABLE_NAME" ] # The name of your Table
+
+store:
+type: FILE
+pathname: "/vanus-connect/data/offset.dat"
+
+db_history_file: "/vanus-connect/data/history.dat"
+EOF
+```
+
```
Comment (ehisakhile) [inline blog/2023-02-19-mysql-email-blog/index.md:154]: Run MySQL Source in the background

```shell
  docker run -it --rm --network=host \
  -v ${PWD}:/vanus-connect/config \
  -v ${PWD}:/vanus-connect/data \
  --name source-mysql public.ecr.aws/vanus/connector/source-mysql &
```

Add Shell Command around code

**Review by ehisakhile — COMMENTED** [review]
Please check comments

**Review by ehisakhile — APPROVED** [review]

File: blog/2023-02-19-mysql-email-blog/index.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
+---
+title: Build a Notification Pipeline from MySQL to Email in Seconds
+dscription: We will explore how to use Vanus to build a pipeline that sends email notifications based on changes in a MySQL database. We will cover the necessary steps to set up Vanus, configure it to monitor a MySQL database, and send email notifications when changes are detected.
```
Comment (Michaelg22) [inline blog/2023-02-19-mysql-email-blog/index.md:3]:
```suggestion
description: We will explore how to use Vanus to build a pipeline that sends email notifications based on changes in a MySQL database. We will cover the necessary steps to set up Vanus, configure it to monitor a MySQL database, and send email notifications when changes are detected.
```

**Pull request merged**